src/client/index.html

```diff
@@ -42,6 +42,30 @@
       <!-- Hero Image goes here -->
 
       <!-- Firefox Download section goes here -->
+      <section id="download-section">
+          <div class="container">
+                <div class="row">
+                    <div class="col-md-8 col-sm-12 container">
+                        
+                        <div class="row">
+                          <h1 class="col-md-4">
+                              <img id="ff-download-img" src="https://mozorg.cdn.mozilla.net/media/img/home/voices/firefox-logo-wordmark-white-high-res.8ba7fd303224.png" class="img-responsive" alt="Firefox Logo" height="190" width="190" />
+                          </h1>
+                          <h1 id="ff-download-message-h" class="col-md-8 text-center">
+                              Committed to <strong>you, your privacy</strong> and an<strong> open web</strong>
+                          </h1>
+                        </div>
+                    </div>
+                    <div class="container col-md-4 col-sm-12">
```

Comment:
I think this is required for responsive design. Is there some other way to do this ?